src/test/java/org/folio/rest/impl/DataExportTest.java

```diff
@@ -75,14 +75,15 @@ public void shouldNotCallInventory_whenuploadFilehasOnlyUnderyingSRS(TestContext
       .compose(fileExportDefinitionOptional -> assertCompletedFileDefinitionAndExportedFile(context, fileExportDefinitionOptional))
       .compose(fileExportDefinition -> jobExecutionDao.getById(fileExportDefinition.getJobExecutionId(), okapiConnectionParams.getTenantId())
         .compose(jobExecutionOptional -> assertSuccessJobExecution(context, fileExportDefinition, jobExecutionOptional, 2))
-        .compose(optional -> validateExternalCalls(context, 1,0))
+        .compose(optional -> validateExternalCalls(context))
         .onComplete(succeeded -> async.complete())
       ));
   }
 
-  private Future<FileDefinition> validateExternalCalls(TestContext context, int countSrs, int countInstance) {
-    context.assertEquals(countSrs, MockServer.getServerRqRsData(HttpMethod.GET, ExternalPathResolver.SRS).size());
-    context.assertEquals(countSrs, MockServer.getServerRqRsData(HttpMethod.GET, ExternalPathResolver.INSTANCE).size());
+  private Future<FileDefinition> validateExternalCalls(TestContext context) {
```

Comment:
I left it the other way becaue we can reuse the method for different tests

Reply:
Let's fix it when having more scenarios to reuse